src/@next/Tag/Tag.tsx

```diff
@@ -17,5 +32,39 @@ export const Tag = ({ children, ...props }: TagProps) => {
       children
     );
 
-  return <TagStyle {...props}>{content}</TagStyle>;
+  const removeButton = onRemove && (
+    <TagRemoveContainerStyle
+      role={'button'}
+      className={'icon-remove'}
+      tabIndex={0}
+      onMouseDown={() => setIsPressed(true)}
+      onBlur={() => setIsPressed(false)}
+      onMouseLeave={() => {
+        setIsPressed(false);
+        setIsHover(false);
+      }}
+      onMouseUp={() => setIsPressed(false)}
+      onMouseEnter={() => setIsHover(true)}
+    >
+      <Icon
+        name={'ri-close'}
+        fill={Neutral.B40}
+        height={'20'}
+        onClick={() => onRemove?.(value)}
```

Comment:
```suggestion
        onClick={onRemove}
```
i think you can just do this

Reply:
i have this error, you got other suggestion on resolving it?

<img width="738" alt="image" src="https://user-images.githubusercontent.com/4475526/215383882-3401ec64-7673-4806-b008-073ddc7a3387.png">

Reply:
i think we could actually just remove the parameter inside the onRemove. `() => void` as from my initial suggestion. i checked polaris, they have the same typing

Reply:
storybook will complain it though

<img width="785" alt="image" src="https://user-images.githubusercontent.com/4475526/215389880-3572ebbe-da95-4186-8d2a-798b62fd8537.png">

Reply:
i think this is done